scss/components/_case-pager.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file
+ * This file contains scss code for civicase case list tables and related styles
```

Comment:
The name of the file, description and class name does not match.
Is this pager only for caselist view? or this will be used for other pages also?

If its a generic one, then the file name should be, `pager.scss`.
Description should not say that its for case list table.

Reply:
Done